rest_framework_json_api/parsers.py

```diff
@@ -4,6 +4,8 @@
 from rest_framework import parsers
 from rest_framework.exceptions import ParseError
 
+from django.conf import settings
```

Comment:
Yeah, the style is a bit of a mess because it's not consistent. Don't worry about it. I created issue #344 after I left this comment above so it can be cleaned up in the future. I'll get this line later.